src/main/java/com/ithillel/Application.java

```diff
@@ -0,0 +1,33 @@
+package com.ithillel;
+
+import com.ithillel.service.appcontext.ApplicationContext;
+import com.ithillel.service.appcontext.PropertiesApplicationContext;
+import com.ithillel.service.textprocessor.TextProcessor;
+import java.io.IOException;
+
+public class Application {
+
+    private ApplicationContext applicationContext = new PropertiesApplicationContext();
```

Comment:
For general approach, it will be better to move the PropertiesApplicationContext initialization to the constructor too;  you might like to keep code in one place;